### DISCUSSION TITLE
Testimonial

### DISCUSSION THREAD

**Discussion by dhruvabhat24** [EddieHubCommunity/LinkFree#10146] [Q&A] [answered]
Is that only me facing this issue 
One my client provided me a testimonial 24 hr before till now it is not displaying in my profile. 
The testimonial issue is #10144

**Comment by ArvindParekh** [comment]
Hey @dhruvabhat24, have you tried checking your testimonials through BioDrop forms  ([Reference](https://www.biodrop.io/docs/how-to-guides/testimonials-forms))? Maybe the toggle for the testimonial you're looking for is set to off?

**Reply by dhruvabhat24** [reply]
No toggle is on the thing is I have received two testimonials among them only one is displaying the second one is not even show

**Comment by eddiejaoude** [comment, marked as answer]
I think testimonials will get added to your profile on next deployment, because they are added via json files

**Reply by dhruvabhat24** [reply]
Okay 🙌

**Reply by eddiejaoude** [reply]
This is something we might change in the future to be done via forms

**Comment by dhruvabhat24** [comment]
When will be the next deployment

**Reply by eddiejaoude** [reply]
It usually happens daily, you can see here https://github.com/EddieHubCommunity/BioDrop/actions/workflows/vercel.yml

**Comment by dhruvabhat24** [comment]
> It usually happens daily, you can see here https://github.com/EddieHubCommunity/BioDrop/actions/workflows/vercel.yml

The latest deployment has been done i.e v2.99.5 but still my recieved testimonial is not showing on Biodrop Profile 
![b1](https://github.com/EddieHubCommunity/BioDrop/assets/122305929/cf13e491-503b-4632-9472-37e9dffacc7c)

![b2](https://github.com/EddieHubCommunity/BioDrop/assets/122305929/84260244-625c-49d0-a1a5-fa6910bb9ae2)

**Reply by eddiejaoude** [reply]
I will take a look, thank you for letting me know

**Reply by eddiejaoude** [reply]
For some reason recent testimonials are not being loaded, I have triggered it manually and it is now in your profile for you to enable - I will research why this is happening

**Reply by dhruvabhat24** [reply]
Ok thank you I have received
I hope you will get a solution for it ☺️
